### ISSUE TITLE
cnctaopt link

### ISSUE THREAD

**Issue by zbluebugz** [netquik/CnCTA-SoO-SCRIPT-PACK#8]
Can you update the "TA_CnCTAOpt_Link_Button.user.js" code, please?  

New version is available from: https://github.com/zbluebugz/CnC-TA-Opt

tia.

**Comment by netquik** [comment]
yes i will include in the pack as i release new version thanks mate

Il giorno dom 31 gen 2021 alle ore 17:34 zbluebugz <notifications@github.com>
ha scritto:
